src/main/java/ladder/domain/ladder/Line.java

```diff
@@ -0,0 +1,64 @@
+package ladder.domain.ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.stream.IntStream.*;
+
+public class Line {
+
+    private final List<Point> points;
+
+    public Line(List<Point> points) {
+        this.points = points;
+    }
+
+    public List<Point> getPoints() {
+        return points;
+    }
+
+    public static Line createLine(int players) {
+        List<Point> points = new ArrayList<>();
+        createFirstPoint(points);
+        createAfterPoint(points, players);
+        return new Line(points);
```

Comment:
```suggestion
        List<Point> points = new ArrayList<>();
        
        createFirstPoint(points);
        createAfterPoint(points, players);
        
        return new Line(points);
```
코드의 가독성을위해 세로 밀집도를 신경써보는것도 좋습니다 ㅎ